### PR TITLE
:zap: [RND-49612] Add theming for InputField and FieldWrapper

### PR DESCRIPTION
**Pull request by chaeMil**
#### Summary

*Describe anything that you feel other developers should know.*

#### Testing steps

*Info about test cases. If you think the issue is not testable, describe why.*

#### Follow-up issues

*Is there some action/cleanup needed after this PR is merged or deployed (e.g. consolidate some part outside the scope of this PR, etc)? Create issues for it with deadline and note them here and in the PR comments.*

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).

### CODE REVIEW

File: optimus/lib/src/field_wrapper.dart [hunk 1]
```diff
@@ -140,16 +141,28 @@ class _FieldWrapper extends State<FieldWrapper> {
     setState(() {});
   }
 
+  OptimusThemeData get _theme => OptimusTheme.of(context);
```
Comment (Gladkowski) [inline optimus/lib/src/field_wrapper.dart:144]: I would replace it with `ThemeGetter` mixin.
`class _FieldWrapper extends State<FieldWrapper> with ThemeGetter `

File: optimus/lib/src/input_field.dart [hunk 2]
```diff
@@ -156,25 +160,33 @@ class _OptimusInputFieldState extends State<OptimusInputField> {
     setState(() {});
   }
 
+  OptimusThemeData get _theme => OptimusTheme.of(context);
```
Comment (Gladkowski) [inline optimus/lib/src/input_field.dart:163]: Same here. I would replace it with mixin.
`class _OptimusInputFieldState extends State<OptimusInputField> with ThemeGetter`

File: optimus/lib/src/input_field.dart [hunk 3]
```diff
@@ -156,25 +160,33 @@ class _OptimusInputFieldState extends State<OptimusInputField> {
...
       case OptimusWidgetSize.medium:
       case OptimusWidgetSize.large:
-        return preset300m.copyWith(color: OptimusLightColors.neutral900);
+        return preset300m.copyWith(color: color);
     }
   }
 
   // ignore: missing_return
   TextStyle get _placeholderTextStyle {
+    final color = _theme.isDark
```
Comment (Gladkowski) [inline optimus/lib/src/input_field.dart:181]: I would replace it with 
```
    final color = theme.isDark
        ? theme.colors.neutral0t64
        : theme.colors.secondaryTextColor;
```

File: optimus/lib/src/input_field.dart [hunk 4]
```diff
@@ -156,25 +160,33 @@ class _OptimusInputFieldState extends State<OptimusInputField> {
     setState(() {});
   }
 
+  OptimusThemeData get _theme => OptimusTheme.of(context);
+
   // ignore: missing_return
   TextStyle get _textStyle {
+    final color = _theme.isDark
```
Comment (Gladkowski) [inline optimus/lib/src/input_field.dart:167]: I would replace it with 
```
              cursorColor: _theme.isDark
                  ? theme.colors.neutral200
                  : theme.colors.basic,
```

**Review by Gladkowski — CHANGES_REQUESTED** [review]

**Review by Gladkowski — CHANGES_REQUESTED** [review]
I would replace all the `OptimusLightColors` and `OptimusDarkColors` to `theme.colors`.

File: optimus/lib/src/input_field.dart [hunk 5]
```diff
@@ -158,23 +162,29 @@ class _OptimusInputFieldState extends State<OptimusInputField> {
 
   // ignore: missing_return
   TextStyle get _textStyle {
+    final color = theme.isDark
+        ? OptimusDarkColors.neutral0
+        : OptimusLightColors.neutral900;
```
Comment (Gladkowski) [inline optimus/lib/src/input_field.dart:167]: I would replace it with `theme.colors.defaultTextColor` neutral1000 is new default text color

**Review by Gladkowski — APPROVED** [review]

File: optimus/lib/src/input_field.dart [hunk 6]
```diff
@@ -158,23 +161,27 @@ class _OptimusInputFieldState extends State<OptimusInputField> {
 
   // ignore: missing_return
   TextStyle get _textStyle {
+    final color =
+        theme.isDark ? theme.colors.neutral0 : theme.colors.neutral1000;
```
Comment (ookami-kb) [inline optimus/lib/src/input_field.dart:165]: Shouldn't it use `defaultTextColor`/`inverseTextColor` as well?

**Pull request merged**